src/test/run-pass/abi-sysv64-arg-passing.rs

```diff
@@ -98,7 +98,7 @@ mod tests {
     #[derive(Copy, Clone)]
     pub struct Floats { a: f64, b: u8, c: f64 }
 
-    #[link(name = "rust_test_helpers")]
+    #[link(name = "rust_test_helpers", kind = "static")]
```

Comment:
Were these changes (and those below) required? If so that may be quite worrisome as this is a breaking change...